### DISCUSSION TITLE
Use of generic attributes (C# 11)

### DISCUSSION THREAD

**Discussion by YBAZAN** [metalama/Metalama#46] [Q&A] [answered]
C# 11 will introduce an interesting feature: [Generic attributes](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-11.0/generic-attributes)

It seems that at the moment, we cannot use it (the `langVersion` property in the project cannot be set at the `preview` value). Can we expect having Metalama 1.0 compatible with it?

**Comment by YBAZAN** [comment, marked as answer]
I answer to my own question. We can set the `MetalamaAllowPreviewLanguageFeatures` to `true` in the csproj file.

**Reply by gfraiteur** [reply]
Ok. Note that using generic attributes as aspects is not supported.
